api/envoy/api/v2/discovery.proto

```diff
@@ -93,3 +93,92 @@ message DiscoveryResponse {
   // required for non-stream based xDS implementations.
   string nonce = 5;
 }
+
+// IncrementalDiscoveryRequest and IncrementalDiscoveryResponse are used in a
+// new GRPC endpoint for Incremental XDS. The feature is not supported for REST
+// management servers.
+//
+// With Incremental XDS, the IncrementalDiscoveryResponses do not need to
+// include a full snapshot of the tracked resources. Instead
+// IncrementalDiscoveryResponses are a diff to the state of a XDS client.
+// In incremental XDS there are per resource versions which allows to track
```

Comment:
Nit: be consistent with whether it is "incremental xDS" or "Incremental xDS".

Reply:
Done.